src/core/Akka.MultiNodeTestRunner/Akka.MultiNodeTestRunner.csproj

```diff
@@ -66,10 +71,6 @@
       <Project>{964F0EC5-FBE6-47C5-8AE6-145114D5DB8C}</Project>
       <Name>Akka.MultiNodeTestRunner.Shared</Name>
     </ProjectReference>
-    <ProjectReference Include="..\Akka.MultiNodeTests\Akka.MultiNodeTests.csproj">
```

Comment:
Removed reference to Akka.MultiNodeTests for the same reason as the individual NodeTestRunner